conf/application-json-logger.xml

```diff
@@ -5,7 +5,8 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
-    <logger name="uk.gov" level="${logger.uk.gov:-INFO}"/>
+    <logger name="uk.gov" level="INFO"/>
```

Comment:
I do not think we should change that?
logger.uk.gov was there to follow the logging best practice https://confluence.tools.tax.service.gov.uk/display/DTRG/The+Life+of+Logs+and+how+to+search+them
With this change, we would have no way to configure logback to see debug logs at all for uk.gov package